scripts/deploy-uk-ocp.py

```diff
@@ -18,8 +18,9 @@ def job():
     try:
         print("executing")
         suffix = get_suffix()
+        # TODO: read the cluster name (without the suffix) from config instead of being hardcoded.
         deploy_cmd = (f"deploy-ocp --ocp4mcoci-conf {environment['ocp_config']}"
-            f" --cluster-name odfcluster-uk-{suffix} --cluster-path /tmp/odfcluster-uk-{suffix}"
+            f" --cluster-name odf-uk-{suffix} --cluster-path /tmp/odfcluster-uk-{suffix}"
```

Comment:
```suggestion
            f" --cluster-name odf-uk-{suffix} --cluster-path /tmp/odf-uk-{suffix}"
```
If the folder name is the same as the cluster name, it will be simple to search for anything inside the cluster folder.

Reply:
But this way is easier to distinguish the name from the install folder (less confusion when looking in the logs): up to the team to decide if they prefer to align the cluster name with the folder.
In any case, it seems this change should be done in the fork they're maintaining
@vbnrh @TimothyAsirJeyasing